src/mesh/index_derivs.cxx

```diff
@@ -2924,7 +2924,7 @@ const Field2D Mesh::indexFDDY(const Field2D &v, const Field2D &f, CELL_LOC outlo
   if ((method == DIFF_SPLIT) || ((method == DIFF_DEFAULT) && (fFDDY == nullptr))) {
     // Split into an upwind and a central differencing part
     // d/dx(v*f) = v*d/dx(f) + f*d/dx(v)
-    return indexVDDY(v, f, outloc, DIFF_DEFAULT) + f * indexDDY(v);
+    return indexVDDY(v, f, outloc, DIFF_DEFAULT) + interp_to(f, outloc) * indexDDY(v);
```

Comment:
Do we need to assert `v.getLocation()== outloc` here or should we pass `outloc` into the last `indexDDY` call?

Reply:
that's a bug, should be passing outloc :+1: